### ISSUE TITLE
[Feature Request] Please add etymology

### ISSUE THREAD

**Issue by navxio** [glja021/cerebro-define#1]
for the record, great plugin!

**Comment by glja021** [comment]
Thanks!! 

Hmm I think the API that the plugin uses at the moment doesn't retrieve etymology information.
If you happen to know another free to use API that does, I'd be happy to give it a try :smile:

**Comment by navxio** [comment]
@glja021 I researched and found out https://developer.oxforddictionaries.com/. This is as powerful as it gets, but users will have to sign up to create their own api keys.. 

Probably will have to configure it via settings etc.

**Comment by glja021** [comment]
Yeah I think it'd improve the plugin, but I'm not sure if most of the users would want to sign up to use it.
Maybe giving an option to provide an api key would make sense?

Yet again, not sure if most people would, I think I'll leave this issue open for now; also they're adding a way to manage plugin's settings from the app, it'd make sense to use that feature, once it's done, to grab the keys.
